pyproject.toml

```diff
@@ -81,7 +81,8 @@ dependencies = [
 [project.optional-dependencies]
 test = [
     "pytest",  # Required for unit tests
-    "pytest-cov"  # Reports on test coverage
+    "pytest-cov",  # Reports on test coverage
+    "pixelmatch"  # Reports on test coverage
```

Comment:
I added the "pixelmatch" requirement to pyproject.toml.

Reply:
Does this belong in the test section or should it be moved to the main dependencies?

Reply:
It is only needed for tests,  if people want to just run,  they don't need it